front-end/src/components/Edit/UserFields.js

```diff
@@ -15,8 +15,7 @@ class UserFields extends Component{
         lastname: '',
         imgurl: '',
         description: '',
-        username: '',
-        theme: ''
```

Comment:
We should need the theme in state for the component

Reply:
Yea all the theme functionality has been re-added.